src/Esfa.Vacancy.Register.Api/App_Start/AutoMapperConfig.cs

```diff
@@ -43,14 +43,14 @@ private static void ApplyAnonymisationToVacancy(DomainTypes.Vacancy src, ApiType
             dest.EmployerDescription = src.AnonymousEmployerDescription;
             dest.EmployerWebsite = null;
 
-            dest.EmployerAddress.AddressLine1 = null;
-            dest.EmployerAddress.AddressLine2 = null;
-            dest.EmployerAddress.AddressLine3 = null;
-            dest.EmployerAddress.AddressLine4 = null;
-            dest.EmployerAddress.AddressLine5 = null;
-            dest.EmployerAddress.PostCode = null;
-            dest.EmployerAddress.Longitude = null;
-            dest.EmployerAddress.Latitude = null;
+            dest.VacancyLocation.AddressLine1 = null;
+            dest.VacancyLocation.AddressLine2 = null;
+            dest.VacancyLocation.AddressLine3 = null;
+            dest.VacancyLocation.AddressLine4 = null;
+            dest.VacancyLocation.AddressLine5 = null;
+            dest.VacancyLocation.PostCode = null;
+            dest.VacancyLocation.Longitude = null;
+            dest.VacancyLocation.Latitude = null;
         }
```

Comment:
The Address object can be mapped in general, no need to be specific as the location is expected all the time. Look at previous check ins for mapping code.